### PR TITLE
Create ${datarootdir}/GeoIP on install. Fixes #29.

### LINKED ISSUE #29: /usr/local/share/GeoIP does not exist
on CentOS 6.6

downloaded and extracted the latest bootstrapped source, then

```
#./configure
#make
#make install
```

execute 

```
$/usr/local/bin/geoipupdate
```

but prompts

```
/usr/local/share/GeoIP does not exist
```

a simple 

```
$mkdir /usr/local/share/GeoIP
```

fixes the problem, perhaps it can be automatically created to avoid such issues?
thanks.

### PR DESCRIPTION
**Pull request by oschwald**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Unreleased
   file. In conjunction with changes to the download service itself, errors
   such as lacking a subscription no longer show up with the message "not a
   valid gzip file".
+* ${datarootdir}/GeoIP is now created on `make install`. Reported by Antonios
```
Comment (horgh) [inline ChangeLog.md:18]: The PR is reporting there will be a merge conflict here. Shall I fix through web interface, or do you prefer otherwise?

Comment (oschwald) [reply ChangeLog.md:18]: Ah, right, Please fix through the web interface if it isn't too difficult. Thanks!

**Review by horgh — CHANGES_REQUESTED** [review]
Looks good other than the ChangeLog conflict!

**Comment by horgh** [comment]
Ah, I'll take care of the conflict. Just saw your reply.

**Comment by oschwald** [comment]
I just resolved it with the web interface. Pretty nifty. Please squash on merge.

**Comment by horgh** [comment]
You are quick!

**Pull request merged**